templates/_worker.js/utils/http.ts

```diff
@@ -40,6 +40,8 @@ export function isUrl(url: string): boolean {
  *
  * Only updates the a parameter if the target does not contain it, or the source value is not empty.
  *
+ * For params prefixed with `nxtP`, it also sets the param without the prefix if it does not exist.
```

Comment:
Could you provide an explanation and a link to this: https://github.com/vercel/next.js/blob/230641adfdc5fd456ca404459026d5957dc69c35/packages/next/src/lib/constants.ts#LL3C22-L3C22
?

Reply:
changed it a little, is that what you meant?

Reply:
Yes, perfect thanks! 😃 🙏

Reply:
(also I just noticed my url was somewhat off 😕 😅 🤷)